### ISSUE TITLE
[Bug]: Attachments are available for unlisted versions with block multiple versions

### ISSUE THREAD

**Issue by ioanarusiczki** [mozilla/addons#15080]
### What happened?

The unlisted display the attachments for block multiple versions

![block multiple versions](https://github.com/user-attachments/assets/9e44c26a-0b13-44a7-b5a6-5c992ffee96a)

### What did you expect to happen?

This is done in admin tools so maybe it could be removed.

### Is there an existing issue for this?

- [X] I have searched the existing issues

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/AMOENG-1134)

**Comment by diox** [comment]
Yeah, since (at the moment) this action redirects and doesn't actually perform an action we should disable the attachment capability on it.
